components/ProfilePage/ProfilePage.js

```diff
@@ -0,0 +1,84 @@
+import gql from 'graphql-tag';
+import { t } from 'ttag';
+import { useQuery } from '@apollo/react-hooks';
+import Head from 'next/head';
+import Container from '@material-ui/core/Container';
+
+import useCurrentUser from 'lib/useCurrentUser';
+
+import AppLayout from 'components/AppLayout';
+import UserPageHeader from './UserPageHeader';
+
+const LOAD_USER = gql`
+  query LoadProfilePage($id: String, $slug: String) {
+    GetUser(id: $id, slug: $slug) {
+      id
+      ...UserHeaderData
+    }
+    ListReplies(filter: { userId: $id }, orderBy: [{ createdAt: DESC }]) {
+      edges {
+        node {
+          id
+          text
+          articleReplies(status: NORMAL) {
+            article {
+              id
+              text
+            }
+          }
+        }
+      }
+    }
+  }
+  ${UserPageHeader.fragments.UserHeaderData}
+`;
+
+function ProfilePage({ id, slug }) {
+  const currentUser = useCurrentUser();
+
+  const { data, loading } = useQuery(LOAD_USER, {
+    variables: { id, slug },
+  });
+
+  const isSelf = currentUser && data?.GetUser?.id === currentUser.id;
```

Comment:
data?.GetUser?.id === currentUser?.id

Reply:
Hmm if both `currentUser` and `data` is nullish (which is probably during the first render), `isSelf` will become true.